modules/util/setEditor.js

```diff
@@ -27,6 +27,7 @@ function setEditor(aScratchpad) {
     filePicker.init(
         GM_util.getBrowserWindow(), EDITOR_PROMPT, nsIFilePicker.modeOpen);
     filePicker.appendFilters(nsIFilePicker.filterApps);
+    filePicker.appendFilter("*.cmd", "*.cmd");
```

Comment:
I suggest this supplement:
``` javascript
    if (Services.appShell.hiddenDOMWindow.navigator.platform
        .indexOf("Win") != -1) {
      filePicker.appendFilter("*.cmd", "*.cmd");
    }
```
And add this line:
https://github.com/greasemonkey/greasemonkey/tree/master/modules/util/setEditor.js#L3:
``` javascript
Components.utils.import("resource://gre/modules/Services.jsm");
```